### PR TITLE
#2865 - Report Builder - Handle accidental absolute paths in column property definitio…

### PR DESCRIPTION
**Pull request by davidjgonzalez**
…ns and a few cosmetic adjustments to Reports CSS.

This stemmed from the issue at: https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/2861 where abs property paths were input into the report builder predicates.

Note: I wanted to add some clientside validation to the GraniteUI Textfield but cannot for the life of me find any OOTB validators, much less an OOTB pattern validator; If anyone know if there is one LMK and ill make another PR catching abs paths on the way in. 

Example outputs w/ changes:

2022-06-22 at 3 15 PM

[All+Asset+Report (9).csv](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/8960715/All%2BAsset%2BReport.9.csv)

### CODE REVIEW

File: bundle/src/main/java/com/adobe/acs/commons/reports/models/ReportCellValue.java [hunk 1]
```diff
@@ -66,6 +67,10 @@ public Object getValue() {
 
   @PostConstruct
   private void init() {
+    if (StringUtils.startsWith(property, "/")) {
```
Comment (klcodanr) [inline bundle/src/main/java/com/adobe/acs/commons/reports/models/ReportCellValue.java:70]: This seems like the same functionality as in your utility, use that here?

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/app-page/clientlibs/css/notifications.less [hunk 2]
```diff
@@ -24,6 +24,6 @@
     top: .5em;
     right: 2em;
     width: 500px;
-    z-index: 10000;
+    z-index: 10000000;
```
Comment (klcodanr) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/app-page/clientlibs/css/notifications.less:27]: image

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/columns/property.jsp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<%--
+  #%L
+  ACS AEM Commons Package
+  %%
+  Copyright (C) 2017 Adobe
```
Comment (klcodanr) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/columns/property.jsp:5]: 2022

**Review by klcodanr — APPROVED** [review]
Some minor tweaks, but looks good to me.

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/columns/property.jsp (hunk 3 above)
Comment (davidjgonzalez) [reply ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/columns/property.jsp]: Also, i was surprised this is only used in the `.../columns/data/results.jsp` that sound right?

I originally broke it out anticipating it being used in several results.jsp

**Pull request merged**